src/main/java/cursor/hw12/model/Node.java

```diff
@@ -0,0 +1,17 @@
+package cursor.hw12.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class Node {
+
+    private int averageSalary;
+    private int frequency;
+
+    public Node(int data) {
+        this.averageSalary = data;
+        this.frequency = 0;
```

Comment:
Why you hard code "0" in the constructor?

Reply:
Resolve